src/pickers/DatePicker.js

```diff
@@ -122,10 +118,8 @@ class openIMISDatePicker extends Component {
             format={secondCalendarFormatting}
             disabled={readOnly}
             value={this.state.value ? this.moveByOneDay(new Date(this.state.value)) : null}
-            {...((!!this.props.minDate || !!disablePast) && {
-              minDate: this.getMinDate(this.props.minDate, disablePast),
-            })}
-            {...(!!this.props.maxDate && { maxDate: this.props.maxDate })}
+            {...(!!this.props.minDate && this.getMinDate())}
```

Comment:
If only disablePast is given as a prop and there is no minDate, would this get added?

Reply:
Not really, thanks for catching that